src/index.ts

```diff
@@ -592,6 +594,10 @@ function resolveRequestDocument(document: RequestDocument): { query: string; ope
   return { query: print(document), operationName }
 }
 
+function callOrIdentity<T>(value: MaybeFunction<T>) {
```

Comment:
Name uses functional programming lingo since I felt that best describes what it does

https://ramdajs.com/docs/#call
https://ramdajs.com/docs/#identity